packages/cli/test/commands/env.test.ts

```diff
@@ -0,0 +1,75 @@
+import fs from 'fs-extra';
+import path from 'path';
+import env from '../../src/commands/env';
+import { setupFixture } from '../helpers/setup-fixture';
+import { client } from '../mocks/client';
+import { defaultProject, useProject } from '../mocks/project';
+import { useTeams } from '../mocks/team';
+import { useUser } from '../mocks/user';
+
+describe('env', () => {
+  describe('pull', () => {
+    it('should handle pulling', async () => {
+      const cwd = setupFixture('vercel-env-pull');
+      useUser();
+      useTeams('team_dummy');
+      useProject({
+        ...defaultProject,
+        id: 'vercel-env-pull',
+        name: 'vercel-env-pull',
+      });
+      client.setArgv('env', 'pull', '--yes', '--cwd', cwd);
+      const exitCode = await env(client);
+      expect(exitCode, client.outputBuffer).toEqual(0);
```

Comment:
I added `client.outputBuffer` as a custom message to failures on these so we'd see what the buffer contained when they fail. Here's an example:

```
$  yarn test test/commands/env.test.ts test/commands/pull.test.ts
yarn run v1.22.15
$ jest test/commands/env.test.ts test/commands/pull.test.ts
 PASS  test/commands/pull.test.ts
 FAIL  test/commands/env.test.ts
  ● env › pull › should handle pulling

    Custom message:
      Downloading "development" Environment Variables for Project vercel-env-pull
    Created .env file [6ms]


    expect(received).toEqual(expected) // deep equality

    Expected: 999
    Received: 0

      21 |       client.setArgv('env', 'pull', '--yes', '--cwd', cwd);
      22 |       const exitCode = await env(client);
    > 23 |       expect(exitCode, client.outputBuffer).toEqual(999);
         |                                             ^
      24 |
      25 |       const rawDevEnv = await fs.readFile(path.join(cwd, '.env'));
      26 |

      at Object.<anonymous> (test/commands/env.test.ts:23:45)

Test Suites: 1 failed, 1 passed, 2 total
Tests:       1 failed, 5 passed, 6 total
Snapshots:   0 total
Time:        1.261 s, estimated 2 s
Ran all test suites matching /test\/commands\/env.test.ts|test\/commands\/pull.test.ts/i.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```